mainnet/shared/genesis_42.json

```diff
@@ -14,23 +14,23 @@
     "londonBlock": 0,
     "mergeForkBlock": 0,
     "terminalTotalDifficulty": 0,
-    "shanghaiTime": 1682612926
+    "shanghaiTime": 1687969198
   },
   "number": "0x0",
   "nonce": "0x1",
   "difficulty": "0x1",
-  "gasLimit": "0x280de80",#42M
+  "gasLimit": "0x280de80",
   "gasUsed": "0x0",
-  "timestamp": "0x644a5a60", #same as MIN_GENESIS_TIME
+  "timestamp": "0x646CE7B0",
   "extraData": "0x416e642074686520776f726c6420626563616d65206f6e65",
   "coinbase": "0x0000000000000000000000000000000000000000",
   "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
   "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
   "alloc": {
-    "0x953534215D31D0f2756b8A855aaFB7E641848c2e": { # FNCE
+    "0x953534215D31D0f2756b8A855aaFB7E641848c2e": {
```

Comment:
11243518/42000000 = 0,2677 => 26.77% != 26.53% as defined on the deposit homepage?

Reply:
this numbers were incorrect. Also thanks for the find about the 330752 LYX that need to be deducted from the LBG balance!! As they are the initial balance on the consensus chain and do not need to be migrated!

Reply:
Checked: this issue was coming from a "rounding" error. The number is not 26.77% but 26.76XXXX% - this is where the diff was. All good.